contributed_definitions/nyaml/NXelectronanalyser.yaml

```diff
@@ -77,6 +86,10 @@ NXelectronanalyser(NXobject):
         spec: ISO 18115-1:2023
         term: 10.15 ff.
         url: https://www.iso.org/obp/ui/en/#iso:std:iso:18115:-1:ed-3:v1:en:term:10.15
+    physical_quantity:
+      enumeration: [length]
+    resolution(NXdata):
```

Comment:
similar

Reply:
Used NX_FLOAT and introduced resolution_errors(NX_FLOAT)